android/src/test/java/com/oblador/keychain/KeychainModuleTests.java

```diff
@@ -150,6 +150,7 @@ public void testFingerprintConfigured_api23() throws Exception {
     //   Fingerprints are configured
     //   fingerprint feature is ignored by android os
     ReactApplicationContext context = getRNContext();
+    shadowOf(context.getPackageManager()).setSystemFeature(PackageManager.FEATURE_FINGERPRINT, true);
```

Comment:
The test was failing without this line, which is present in another test. I assumed that it was a miss from some previous change and added this.